recipes/pygod/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "pygod" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pygod-{{ version }}.tar.gz
+  sha256: 801479cf290b9133c6dc7e9782563df7cae621dd337119ba2c3bb590182dafc5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools >=38.6.0
+  run:
+    - numpy >=1.19.4
+    - networkx
+    - python >=3.7
+    - pytorch >=1.10.0
+    - pytorch_geometric >=2.0.3
+    - scikit-learn >=0.22.1
+    - scipy >=1.5.2
+    - setuptools >=50.3.1.post20201107
+
+test:
+  imports:
+    - pygod
+    - pygod.models
+    - pygod.utils
+  commands:
+    - pip list | grep -E "{{ name }}\s+{{ version }}"
```

Comment:
You don't need this. Leaving the `pip check` commented out is fine. That will help us remember to re-enable it when upstream is fixed.